modules/cluster/outputs.tf

```
@@ -9,6 +9,13 @@ output "certificate_authority_data" {
   depends_on  = [null_resource.cluster_launch]
 }
 
+output "cluster_security_group_id" {
+  description = "The cluster security group that was created by Amazon EKS for the cluster"
+  value       = aws_eks_cluster.cluster.vpc_config.0.cluster_security_group_id
```

So looking into this line, while it works, it looks like this is no longer the [desired syntax](https://www.terraform.io/docs/configuration/expressions.html#references-to-resource-attributes) for accessing list items, evidently even with splat syntax.  Normally would not really care, but since its the line causing the linting failure, I think we should probably just bump it to the newer format.  At least locally, this does seem to pass linting.


```suggestion
  value       = aws_eks_cluster.cluster.vpc_config[0].cluster_security_group_id
```